charts/ccd-data-store-api/values.preview.template.yaml

```diff
@@ -16,7 +16,7 @@ java:
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
     DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
-    USER_PROFILE_HOST: http://ccd-user-profile-api-pr-224.service.core-compute-preview.internal/
+    USER_PROFILE_HOST: http://ccd-user-profile-api-pr-302.service.core-compute-preview.internal/
```

Comment:
why we are using pr-302 here?

Reply:
Because pr-224 is no longer present in preview-01-rg, and pr-302 is. Otherwise Smoke tests were failing, when fetching the user profile.